test/offchainAsset/OffChainAssetReceiptVault.test.ts

```diff
@@ -8,7 +8,7 @@ import {
   fixedPointMul,
```

Comment:
how am i supposed to review this file? there's 2000 lines changed and most are unrelated to the conversion